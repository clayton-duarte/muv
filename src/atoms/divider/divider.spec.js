import React from 'react';

import Divider from '.';

it('should render a divider', () => {
  const wrapper = mount(
    <Divider />,
  );
  expect(wrapper).toMatchSnapshot();
});
