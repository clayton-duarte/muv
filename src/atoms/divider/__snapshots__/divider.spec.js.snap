// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a divider 1`] = `
<Divider>
  <styled.hr
    theme={
      Object {
        "animation": Object {
          "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
          "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
          "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
          "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
          "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
        },
        "color": Object {
          "black": "#222222",
          "darkGray": "#515151",
          "error": "#dd3333",
          "gray": "#a9abae",
          "lightGray": "#e5e5e5",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0081c8",
          "secondary": "#00245f",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "font": Object {
          "color": "#222222",
          "family": "\\"Ubuntu\\", sans-serif",
          "size": "16px",
          "spacing": 0,
        },
        "shape": Object {
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "maxWidth": "1280px",
          "padding": ".75rem .875rem",
          "radius": ".5rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
          "size": "16px",
        },
        "transition": Object {
          "time": ".3s ease",
        },
      }
    }
  >
    <StyledComponent
      forwardedClass={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": false,
            "lastClassName": "eLrOJG",
            "rules": Array [
              "
border: ",
              [Function],
              ";
margin: ",
              [Function],
              ";
border-right: 0;
border-left: 0;
border-top: 0;
",
            ],
          },
          "defaultProps": Object {
            "theme": Object {
              "animation": Object {
                "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
                "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
                "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
                "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
                "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
              },
              "color": Object {
                "black": "#222222",
                "darkGray": "#515151",
                "error": "#dd3333",
                "gray": "#a9abae",
                "lightGray": "#e5e5e5",
                "overlay": "rgba(0, 0, 0, .15)",
                "primary": "#0081c8",
                "secondary": "#00245f",
                "success": "#33bb33",
                "warn": "#dd3333",
                "white": "#fefefe",
              },
              "font": Object {
                "color": "#222222",
                "family": "\\"Ubuntu\\", sans-serif",
                "size": "16px",
                "spacing": 0,
              },
              "shape": Object {
                "border": ".125rem solid rgba(0, 0, 0, .15)",
                "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
                "margin": ".25rem .125rem",
                "maxWidth": "1280px",
                "padding": ".75rem .875rem",
                "radius": ".5rem",
                "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                "size": "16px",
              },
              "transition": Object {
                "time": ".3s ease",
              },
            },
          },
          "displayName": "styled.hr",
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "hr",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      theme={
        Object {
          "animation": Object {
            "bounce": "
  @keyframes bounce {
    from {
      bottom: 0;
    }
    25% {bottom: .5rem;}
    to {
      bottom: 0
    }
  }
  animation: bounce .3s ease
",
            "fadeIn": "
  @keyframes fadeIn {
    from {
      opacity: 0;
    }
    
    to {
      opacity: 1;
    }
  }
  animation: fadeIn .3s ease
",
            "fadeOut": "
  @keyframes fadeOut {
    from {
      opacity: 1;
    }
    
    to {
      opacity: 0;
    }
  }
  animation: fadeOut .3s ease
",
            "popIn": "
  @keyframes popIn {
    from {
      transform: scale(0);
    }
    
    to {
      transform: scale(1);
    }
  }
  animation: popIn .3s ease
",
            "popOut": "
  @keyframes popOut {
    from {
      transform: scale(1);
    }
    
    to {
      transform: scale(0);
    }
  }
  animation: popOut .3s ease
",
          },
          "color": Object {
            "black": "#222222",
            "darkGray": "#515151",
            "error": "#dd3333",
            "gray": "#a9abae",
            "lightGray": "#e5e5e5",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#0081c8",
            "secondary": "#00245f",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "font": Object {
            "color": "#222222",
            "family": "\\"Ubuntu\\", sans-serif",
            "size": "16px",
            "spacing": 0,
          },
          "shape": Object {
            "border": ".125rem solid rgba(0, 0, 0, .15)",
            "float": "0 .5rem .5rem .125rem rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "maxWidth": "1280px",
            "padding": ".75rem .875rem",
            "radius": ".5rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            "size": "16px",
          },
          "transition": Object {
            "time": ".3s ease",
          },
        }
      }
    >
      <hr
        className="sc-bdVaJa eLrOJG"
      />
    </StyledComponent>
  </styled.hr>
</Divider>
`;
